src/Library/Inputs/GrpcInput/GrpcInput.cs

```diff
@@ -138,6 +148,8 @@ public InputStats GetStats()
             catch (TaskCanceledException)
             {
                 // we have been stopped
+                Diagnostics.LogError("cancel batch");
```

Comment:
This means we have been cancelled from the outside, I suggest we don't log an error. Not a big deal to load, but it's a bit confusing to see an error in the logs in the event of the channel being stopped.

Reply:
oh, sorry this is my debug output, will clean it up